src/components/serverNotifications/serverNotifications.js

```diff
@@ -197,30 +197,30 @@ define(['connectionManager', 'playbackManager', 'events', 'inputManager', 'focus
         events.on(apiClient, "message", onMessageReceived);
     }
 
-    function enableNativeGamepadKeyMapping() {
-        if (window.navigator && "string" == typeof window.navigator.gamepadInputEmulation) {
-            window.navigator.gamepadInputEmulation = "keyboard";
-            return true;
+    function attachgamepad(e) {
+        if (navigator.getGamepads.length > 0) {
+            require(["components/serverNotifications/gamepadtokey"]);
+            console.log("Gamepad connected! Attaching gamepadtokey.js script");
         }
-
-        return false;
     }
 
-    function isGamepadSupported() {
-        return "ongamepadconnected" in window || navigator.getGamepads || navigator.webkitGetGamepads;
+    function dettachgamepad(e) {
+        delete require.cache[require(["components/serverNotifications/gamepadtokey"])];
+        console.log("Gamepad disconnected! Dettaching gamepadtokey.js");
     }
 
     connectionManager.getApiClients().forEach(bindEvents);
-
     events.on(connectionManager, 'apiclientcreated', function (e, newApiClient) {
         bindEvents(newApiClient);
     });
-
-    if (!enableNativeGamepadKeyMapping() && isGamepadSupported()) {
+    if (navigator.getGamepads()[0] != null) {
         require(["components/serverNotifications/gamepadtokey"]);
+        console.log("Gamepad connected! Attaching gamepadtokey.js script");
+        window.addEventListener("gamepaddisconnected", dettachgamepad);
+    } else {
+        window.addEventListener("gamepadconnected", attachgamepad);
+        console.log("No gamepad connected to this device");
     }
-
-    require(["components/serverNotifications/mouseManager"]);
-
+    // require(["components/serverNotifications/mouseManager"]);
```

Comment:
why?

Reply:
As I specify in the PR, it doesn't seem to me that mousemanager.js does something useful. Do you know its purpose?

If we are sure that it's useful, we can keep it, otherwise I think we should get rid of it in this PR.

Reply:
cc @dmitrylyzo

Reply:
`mouseManager` detects mouse inactivity - to hide the cursor (adds CSS classes). Also, this beast autofocus on hover in Tizen - it is pretty annoying (it’s good that I don’t use the mouse on the TV).

Reply:
Worth removing then? Or perhaps just check if the browser is a TV before firing it like what I want to do with ``gamepadtokey.js``?

Reply:
I think this is also essential for non-TV devices.
From what I noticed (if you disable `mouseManager`):
* The cursor does not disappear (even in browser in non-TV layout) - it’s bad when you watch a video or pictures. In the Tizen emulator, cursor is hided by system itself.
* In TV layout in browser, volume control is displayed (normally it is available only on mouse move) - not so bad, but the slider itself does not focus, so as not to get stuck in it when keyboard navigating.
* In TV layout, when viewing pictures, the navigation buttons do not disappear.